lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_reopening.py

```diff
@@ -0,0 +1,1312 @@
+                                            encounter_reference: dict) -> Optional[dict]:
+    """
+    Returns a FHIR Questionnaire Response resource entry for the
+    testing_determination_internal instrument.
+    """
+
+    string_questions = [
+        'testing_trigger',
+        'testing_type'
+    ]
+
+    question_categories = {
+        'valueString': string_questions
+    }
+
+    return questionnaire_response(record, question_categories, patient_reference, encounter_reference, True)
+
+def create_computed_questionnaire_response(record: dict, patient_reference: dict,
+                                            encounter_reference: dict,
+                                            birthdate: datetime, encounter_date: datetime) -> Optional[dict]:
+    """
+    Returns a FHIR Questionnaire Response resource entry for a
+    computed questionnaire.
+    """
+
+    record['age'] = relativedelta(encounter_date, birthdate).years
+
+    integer_questions = [
+        'age'
+    ]
+
+    question_categories = {
+        'valueInteger': integer_questions
+    }
+
+    return questionnaire_response(record, question_categories, patient_reference, encounter_reference)
+
+def create_daily_questionnaire_response(record: dict, patient_reference: dict,
+                                            encounter_reference: dict) -> Optional[dict]:
+    """
+    Returns a FHIR Questionnaire Response resource entry for the daily
+    attestation questionnaire.
+    """
+    boolean_questions = [
+        'daily_symptoms',
+        'prev_pos',
+        'hall_health_2',
+        'prior_test',
+        'travel_countries_phs',
+        'travel_states_phs',
+        'screen_positive'
+    ]
+
+    string_questions = [
+        'daily_symptoms_covid_like',
+        'daily_exposure',
+        'daily_exposure_known_pos',
+        'symptoms',
+        'sought_care',
+        'hospital_where',
+        'hospital_ed',
+        'prior_test_positive',
+        'prior_test_type',
+        'prior_test_result',
+        'contact',
+        'wash_hands',
+        'clean_surfaces',
+        'hide_cough',
+        'mask',
+        'distance',
+        'attend_event',
+        'wfh',
+        'country1',
+        'country2',
+        'country3',
+        'country4',
+        'country5',
+        'state1',
+        'state2',
+        'state3',
+        'state4',
+        'state5',
+        'countries_visited',
+        'states_visited'
+    ]
+
+    date_questions = [
+        'attestation_timestamp',
+        'symptom_onset',
+        'hospital_arrive',
+        'hospital_leave',
+        'prior_test_positive_date'
+    ]
+
+    question_categories = {
+        'valueBoolean': boolean_questions,
+        'valueString': string_questions,
+        'valueDate': date_questions
+    }
+
+    # Combine checkbox answers into one list
+    checkbox_fields = [
+        'daily_symptoms_covid_like',
+        'symptoms',
```

Comment:
No need to ingest symptoms here since it's already part of encounter.

Reply:
Same as above. Ingesting them as questionnaire items for convenience.

Reply:
I'm no longer ingesting the symptom questions as questionnaire items.